### ISSUE TITLE
NotFound: No such file or directory

### ISSUE THREAD

**Issue by whoizit** [eliaperantoni/Koi#10]
```
> rustc --version
rustc 1.54.0-nightly (c1e8f3a58 2021-05-30)
```
```
> cargo install --git https://github.com/eliaperantoni/Koi
...
warning: 3 warnings emitted

    Finished release [optimized] target(s) in 2m 23s
  Installing /home/user/.asdf/installs/rust/nightly/bin/koi
   Installed package `koi v0.1.0 (https://github.com/eliaperantoni/Koi#caa13f7b)` (executable `koi`)
```
```
/home/user/tmp/koi/examples(master)> asdf reshim
/home/user/tmp/koi/examples(master)> koi
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/main.rs:67:75
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
/home/user/tmp/koi/examples(master)> koi import.koi
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/interp/mod.rs:150:60
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by whoizit** [comment]
strange
```
/home/user(master)> koi tmp/koi/examples/import.koi
4
5
/home/user(master)> cd tmp/koi/examples/
/home/user/tmp/koi/examples(master)> koi import.koi
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/interp/mod.rs:150:60
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by whoizit** [comment]
```
~/build/koi/examples > RUST_BACKTRACE=1 ../target/debug/koi import.koi
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/interp/mod.rs:150:60
stack backtrace:
   0: rust_begin_unwind
             at /rustc/e4a60327063e82413eed50a10df3b7d19b77bda0/library/std/src/panicking.rs:515:5
   1: core::panicking::panic_fmt
             at /rustc/e4a60327063e82413eed50a10df3b7d19b77bda0/library/core/src/panicking.rs:92:14
   2: core::result::unwrap_failed
             at /rustc/e4a60327063e82413eed50a10df3b7d19b77bda0/library/core/src/result.rs:1355:5
   3: core::result::Result<T,E>::unwrap
             at /rustc/e4a60327063e82413eed50a10df3b7d19b77bda0/library/core/src/result.rs:1037:23
   4: koi::interp::Interpreter::run_stmt
             at /home/user/build/koi/src/interp/mod.rs:150:30
   5: koi::interp::Interpreter::run
             at /home/user/build/koi/src/interp/mod.rs:55:13
   6: koi::main
             at /home/user/build/koi/src/main.rs:78:5
   7: core::ops::function::FnOnce::call_once
             at /rustc/e4a60327063e82413eed50a10df3b7d19b77bda0/library/core/src/ops/function.rs:227:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

**Comment by eliaperantoni** [comment]
Hi @whoizit, the first call to `koi` fails because it expects a source file path when `-s` is not used.
The other errors are 100% legit and I hope to have fixed them in release v1.7.1.

```
~/code/koi > koi examples/import.koi 
4
5
~/code/koi > cd examples/
~/code/koi/examples > koi import.koi 
4
5
~/code/koi/examples > cd /
/ > koi /home/elia/code/koi/examples/import.koi 
4
5
```

I'm closing the issue for now but feel free to reopen it if you encounter more issues with the import system.
